getting-started/data-node-authentication.md

```diff
@@ -0,0 +1,99 @@
+node have the necessary roles and permissions on all the data nodes
+where distributed hypertables are placed.
+
+This is critical since when TimescaleDB is executing a query or
+command it will use the same role on the data nodes as on the access
+node. This means that the same roles need to exist on the data nodes
+and the access node need to have authentication set up for those
+roles.
+
+In a typical setup, you will create the distributed hypertable in a
+database and use the same database on all the data nodes. This means
+that you need to ensure that a user that is connected to the access
+node can access the same databases on the data nodes.
+
+>:NOTE: Setting up a secure system is a complex task and this section
+> should not be read as recommending any particular security measures
+> for securing your system. The chapter contains technical
+> instructions for how to set up the different authentication methods.
+
+## Setting up Password Authentication [](distdb-auth.pass)
+
+The easiest authentication to set up is password
+authentication. Configuring password authentication between the access
+node and data nodes is a two step process:
+
+1. Each data node needs to be configured to accept and authenticate
+   connections using passwords.
+
+2. The access node needs data node passwords for each role using the
+   access node. We will deal with each step separately below.
+
+In this section, the focus is on setting up SCRAM SHA-256 password
+authentication. For other password authentication methods, [see the
+PostgreSQL manual][auth-password].
+
+>:NOTE: The `password` method should not be used since it sends the
+> password in cleartext, and the `md5` is using a cryptographic hash
+> function that is no longer considered secure.
+
+### Setting up Password Authentication on the Data Node [](distdb-auth.pass.data-node)
+
+In order to set up password authentication for a user `new_user`, it
+is necessary to update the HBA (Host-Based Authentication) file on the
+data nodes to enable password authentication for `new_user`. The
+default name of the HBA file is `pg_hba.conf`. Details of [the
+`pg_hba.conf` format can be found in the PostgreSQL
+manual.][postgresql-hba].
+
+In this case, you want to give `new_user` in the `example.com` domain
+access to all databases on the data node, so add the following line to
+the `pg_hba.conf`.
+
+```
+host    all    new_user    .example.com    scram-sha-256
+```
+
+Once you have set up the authentication method on the data node, you
+need to check the configuration file (normally `postgresql.conf` in
+the data directory of the data node server) and make sure that the
+`hba_file` option is correctly set.
+
+### Setting up Passwords on the Access Node [](distdb-auth.pass.access-node)
+
+To execute queries across data nodes, the access node needs to connect
+to data nodes as the user executing the query. This means that the
+access node needs passwords for each data node, which are stored in a
```

Comment:
```suggestion
access node needs passwords for each user, which are stored in a
```

Reply:
Hmmmm, it is not clear if it is a user on a data node or on the access node. Will take a shot.

Reply:
Done. PTAL.